docs/gethue/content/en/posts/2023-01-23-release-hue-4.11.md

```diff
@@ -0,0 +1,140 @@
+We’ve also updated the autocomplete and syntax checker for Hive and Impala to support the latest Iceberg syntax.
+
+Read more about the [Creating Iceberg tables in Hue](https://gethue.com/blog/2022-10-11-creating-iceberg-tables-in-hue/).
+
+![Iceberg Hue integration](https://cdn.gethue.com/uploads/2022/10/iceberg1.gif)
+
+
+#### SparkSQL Improvements
+
+Hue leverages Apache Livy 3 to support Spark SQL queries in Hue on the Apache Spark 3 engine. Since booting a new Livy session was slow (around 30-45s), we are now caching the session details per user level for faster query experience. 
+
+This also helps in database/table/column autocomplete in the left and right assist.
+
+We also show full table details in the Table Browser. Hue now cleans up unused Livy sessions as well.
+
+We now have a dedicated autocomplete and syntax checker available for Spark SQL! It supports all Spark SQL statement types up to date with version 3.3.1.
+
+On top of this we’ve also included the UDF library for Spark SQL, accessible in the autocomplete as well as the right assist panel. It covers documentation of the built in functions of Spark SQL 3.3.1.
+
+![functioning SparkSQL query editor](https://cdn.gethue.com/uploads/2023/01/sparksql_left_right_assist.gif)
+
+
+#### HPL/SQL Support
+
+HPL/SQL is an Apache open source procedural extension for SQL for Hive users. It has its own grammar. It is included with Apache Hive from version 2.0. 
+
+You can enable the HPL/SQL dialect via desktop/conf/hue.ini config file section.
+        
+        [notebook]
+        [[interpreters]]
+        [[[hplsql]]]
+        name=Hplsql
+        interface=hiveserver2
+
+Read more about the [HPL/SQL Support](https://gethue.com/blog/2022-02-01-hplsql-support/).
+
+![Example](https://cdn.gethue.com/uploads/2022/02/Hplsql_example1.png)
+
+
+#### Tech stack & Tooling
+
+- [Create SQL tables from excel files](https://gethue.com/blog/2021-11-15-create-sql-tables-from-execl-files/)
+- [Access your data in ABFS without any credential keys](https://gethue.com/blog/2021-09-21-access-your-data-in-abfs-without-any-credential-keys/)
+- [Create Phoenix tables in Just 2 steps](https://gethue.com/blog/2021-08-17-create-phoenix-tables-in-just-2-steps/)
+- [Open In Importer and Copy Path Options in Filebrowser](https://gethue.com/blog/2021-08-10-open-in-importer-and-copy-path-options-in-filebrowser/)
+- [Create SQL tables on the fly with zero clicks](https://gethue.com/blog/2021-07-26-create-sql-tables-on-the-fly-with-zero-clicks/)
+- [Azure Storage sharing by leveraging SAS tokens so that your users don’t need credentials](https://gethue.com/blog/2021-06-30-how-to-use-azure-storage-rest-api-with-shared-access-sginature-sas-tokens/)
+
+
+It has more than 650+ commits and 100+ bug fixes! For more details on all the changes, check out the [release notes](https://docs.gethue.com/releases/release-notes-4.11.0/).
```

Comment:
This release of Hue has 650+ commits and 100+ bug fixes. For a complete list, see the [release notes](https://docs.gethue.com/releases/release-notes-4.11.0/).